src/controllers/user.js

```diff
@@ -58,12 +59,34 @@ export const getAll = async (req, res) => {
 
 export const updateById = async (req, res) => {
   const { cohort_id: cohortId } = req.body
+  const userToUpdateId = Number(req.params.id)
+  try {
+    const userToUpdate = await User.findById(userToUpdateId)
+    const foundCohort = await Cohort.findCohortByID(cohortId)
+    const updatedProfile = await userToUpdate.update()
+    userToUpdate.cohortId = cohortId
 
-  if (!cohortId) {
-    return sendDataResponse(res, 400, { cohort_id: 'Cohort ID is required' })
+    if (!userToUpdate) {
+      return sendDataResponse(res, 400, { message: 'User does not exist' })
+    }
+    if (!cohortId) {
+      return sendDataResponse(res, 400, { message: 'Cohort ID is required' })
+    }
+    if (typeof cohortId !== 'number') {
+      return sendDataResponse(res, 400, {
+        message: 'Cohort ID must be a integer'
+      })
+    }
+    if (foundCohort === null) {
+      return sendDataResponse(res, 400, {
+        message: 'Cohort could not be found'
+      })
+    }
+    return sendDataResponse(res, 200, { test: updatedProfile })
```

Comment:
Is `test` the expected response body key?

Reply:
We forgot to adjust this, we have refactored to get the expected response in the data response